web_src/js/features/repo-projects.ts

```diff
@@ -1,31 +1,31 @@
-import $ from 'jquery';
 import {contrastColor} from '../utils/color.ts';
 import {createSortable} from '../modules/sortable.ts';
 import {POST, DELETE, PUT} from '../modules/fetch.ts';
+import {fomanticQuery} from '../modules/fomantic/base.ts';
 
-function updateIssueCount(cards) {
-  const parent = cards.parentElement;
+function updateIssueCount(card: HTMLElement): void {
+  const parent = card.parentElement;
   const cnt = parent.querySelectorAll('.issue-card').length;
-  parent.querySelectorAll('.project-column-issue-count')[0].textContent = cnt;
+  parent.querySelectorAll('.project-column-issue-count')[0].textContent = String(cnt);
```

Comment:
If would suggest to really rewrite, why `parent.querySelectorAll('.project-column-issue-count')[0]`?